src/server.cpp

```diff
@@ -437,6 +440,52 @@ Server::~Server()
 	}
 }
 
+IRollbackManager *Server::createRollbackManager()
+{
+	Settings world_mt;
+	const std::string world_mt_path = m_path_world + DIR_DELIM + "world.mt";
+	// If the world config can't be read for some reason, fall back to defaults below.
```

Comment:
This indicates an incomplete setup or data corruption. I would suggest to raise an exception like in `Server::openModStorageDatabase` or at least log an error like in `Server::migrateModStorageDatabase`.